common/store/store.js

```diff
@@ -240,6 +240,18 @@ export class StoreSyncedWithLocalStorageProvider extends React.Component {
       : objectList[0].id;
   };
 
+  addToEntityObject = (objects, entity) => {
```

Comment:
This method was added to allow to add more entries in the "missions" entity without removing the previous ones.